packages/language-server/src/completion-items.ts

```diff
@@ -54,6 +54,9 @@ function transformToLspSuggestions(
   const lspSuggestions = map(suggestions, suggestion => {
     const lspKind = computeLSPKind(suggestion);
     let detailText = getNodeDetail(suggestion.ui5Node);
+    if (suggestion.ui5Node.experimentalInfo?.isExperimental) {
+      detailText = `(experimental) ${detailText}`;
```

Comment:
Did you consult with Yuval on whether we want this? I'm not sure we do...
Also, if we do, the "(experimental)" should be after the "(deprecated)" (or maybe not be there at all for deprecated APIs - ask Yuval this, too, please).

Reply:
It was verified with Yuval, "(experimental)" should be after the "(deprecated)"

Reply:
should not we use Tags for deprecated instead of modifying the description?
- 
![image](https://user-images.githubusercontent.com/4233843/79777796-75757900-8340-11ea-985f-79144568a525.png)

Reply:
https://microsoft.github.io/language-server-protocol/specifications/specification-current/

Reply:
We would but it causes crashes in Theia (it's not only unsupported but actually causes the code assist to not work). There's a comment about it where we create the `CompletionItem` instance.
This is the comparison between VSCode and Theia features:
https://che-incubator.github.io/vscode-theia-comparator/status.html